core/lib/zksync_core/src/state_keeper/updates/miniblock_updates.rs

```diff
@@ -59,13 +59,21 @@ impl MiniblockUpdates {
         }
     }
 
-    pub(crate) fn extend_from_fictive_transaction(&mut self, result: VmExecutionResultAndLogs) {
+    pub(crate) fn extend_from_fictive_transaction(
+        &mut self,
+        result: VmExecutionResultAndLogs,
+        l1_gas_count: BlockGasCount,
+        execution_metrics: ExecutionMetrics,
```

Comment:
Bikshedding: Couldn't you instantiate `ExecutionMetricsForCriteria::new(None, &result)` directly inside this method to compact the things even further? Specifically, it would signify that `MiniblockUpdates` metrics are updated based on `result` only.

Reply:
I'd like to keep this way to be consistent between `extend_from_fictive_transaction` and `extend_from_executed_transaction`. Maybe this whole part worth a refactoring, but it shouldn't be a part of this PR